src/io/mod.rs

```diff
@@ -12,7 +12,10 @@ use thiserror::Error;
 mod stun;
 pub(crate) use stun::stun_resend_delay;
 pub use stun::StunMessage;
-pub(crate) use stun::{StunError, TransId, STUN_MAX_RETRANS, STUN_MAX_RTO_MILLIS, STUN_TIMEOUT};
+pub(crate) use stun::{
+    Class as StunClass, Method as StunMethod, StunError, TransId, STUN_MAX_RETRANS,
```

Comment:
I thought their original names would cause too much confusion when used in outside contexts like `IceAgent`, so I renamed them at re-export time. Would it perhaps be better to just rename the original enums?

Reply:
Looks good